Code/app/Http/Controllers/ResourceController.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Resource;
+use Illuminate\Http\Request;
+
+
+
+
+/*
+
+TODO:
+Think about each of the functions we need in each of our controllers. 
+
+Implement the code using the models.
+We want to be able to crate a resource that can be manipulated at will. What do we want in this resource?
+
+Use functions 
+
+
+
+*/
+
+
+
+class ResourceController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $newResource = new Resource;
+        $newResource->name = $request->name;
+        $newResource->author_id = $request->author_id;
+        $newResource->use_id = $request->use_id;
+
+        $newResource->save();
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  \App\Resource  $resource
+     * @return \Illuminate\Http\Response
+     */
+    public function show(Resource $resource)
+    {
+        return view('resource', ['resource' fdskoafkeiopsajefiwaojfeiowajfeopwuaj8feo[waujf8eioajfo]]    )
```

Comment:
change this so that you're passing the resource name into resource.blade.php as a variable
you can get the name of the resource by doing `$resource->name`